packages/preview-middleware-client/src/adp/quick-actions/table-quick-action-base.ts

```diff
@@ -86,7 +86,7 @@ export abstract class TableQuickActionDefinitionBase {
         protected readonly defaultTextKey: string,
         protected readonly context: QuickActionContext,
         protected includeServiceAction?: boolean
-    ) {}
+    ) { }
```

Comment:
This seems to be incorrectly formatted.

Reply:
i use auto formatting which always add this space. now i removed manually.